src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -175,28 +175,21 @@ export function DiscoverField({
   return (
     <>
       <EuiPopover
+        ownFocus
```

Comment:
Adding `ownFocus` makes the keyboard navigation way easier and ensures that the popover is closed before moving on to the next when if using keyboard navigation